qt/pyproject.toml

```diff
@@ -0,0 +1,80 @@
+[project]
+name = "aqt"
+# dynamic = ["version"]
+version = "0.1.2"
+requires-python = ">=3.9"
+license = "AGPL-3.0-or-later"
+dependencies = [
+  "beautifulsoup4",
+  "flask",
+  "flask_cors",
+  "jsonschema",
+  "requests",
+  "send2trash",
+  "waitress>=2.0.0",
+  "psutil; sys.platform == 'win32'",
+  "pywin32; sys.platform == 'win32'",
+  "pip-system-certs!=5.1",
+  "mock",
+  "types-decorator",
+  "types-flask",
+  "types-flask-cors",
+  "types-markdown",
+  "types-waitress",
+  "types-pywin32",
```

Comment:
@dae Shouldn't `types-*` packages should be optional development dependencies? Looks like there are potentially some other packages in the `pyproject.toml` files that can be excluded in the installation (mock and werkzeug maybe?).